rust/automerge/src/read.rs

```diff
@@ -129,6 +129,16 @@ pub trait ReadDoc {
     /// Get the type of this object, if it is an object.
     fn object_type<O: AsRef<ExId>>(&self, obj: O) -> Result<ObjType, AutomergeError>;
 
+    /// Get all marks on a current sequence
+    fn get_marks<O: AsRef<ExId>>(&self, obj: O) -> Result<Vec<Mark<'_>>, AutomergeError>;
```

Comment:
We've used `x()` rather than `get_x()` for getters in this trait (which is common in Rust). Suggest we do that here as well.